css/bourbon/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.2.2
+MAINTAINER Joe Ferris <jferris@thoughtbot.com>
+RUN mkdir -p /opt/include
+RUN mkdir /src
+RUN mkdir /build
+RUN gem install sass bourbon
```

Comment:
Small thing -- what about:

```
RUN gem install sass
RUN gem install bourbon -v 4.2.3
```

That way we can bump the bourbon version to trigger
it to be reinstalled, but keep the previous steps cached?